functions/Get-DbaErrorLogConfig.ps1

```diff
@@ -25,7 +25,7 @@ function Get-DbaErrorLogConfig {
 
             Website: https://dbatools.io
             Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
-            License: GNU GPL v3 https://opensource.org/licenses/GPL-3.0
+            License: MIT https://opensource.org/licenses/MIT
```

Comment:
My bad, got this from template and didn't catch it was old. I'll go update the templates.

Reply:
ty!